src/Processors/Formats/Impl/ArrowBufferedStreams.cpp

```diff
@@ -5,14 +5,17 @@
 #include "ArrowBufferedStreams.h"
 
 #if USE_ARROW || USE_ORC || USE_PARQUET
-
+#include <IO/RemoteReadBufferCache.h>
 #include <Common/assert_cast.h>
 #include <IO/ReadBufferFromFileDescriptor.h>
 #include <IO/WriteBufferFromString.h>
 #include <IO/copyData.h>
 #include <arrow/buffer.h>
 #include <arrow/io/memory.h>
 #include <arrow/result.h>
+#include <base/logger_useful.h>
+#include <Common/Stopwatch.h>
+#include <Poco/Logger.h>
```

Comment:
Don't see why we need these new headers here

Reply:
Removed